config/jobs/image-pushing/releng/k8s-staging-releng.yaml

```
@@ -77,6 +77,7 @@ postsubmits:
         testgrid-dashboards: sig-release-releng-informing, sig-release-image-pushes
         testgrid-alert-email: release-managers+alerts@kubernetes.io
       decorate: true
+      run_if_changed: '^images\/releng\/k8s-ci-builder\/'
```

Add this to run only if the image directory change, or there is a reason we want to build every time?

---

```suggestion
      run_if_changed: '^(cmd\/|compile-release-tools|go.(mod|sum)|images\/releng\/k8s-ci-builder\/|pkg\/)'
```

Yes, k8s-ci-builder builds all tools in k/release using `./compile-release-tools`: https://github.com/kubernetes/release/blob/fce32c9696f0be229e9898e51cbbe8e497e72027/images/releng/k8s-ci-builder/Dockerfile#L21-L23

...which means any change to the repo packages should be trigger a rebuild.

ref: https://regex101.com/r/cZIITF/1/

---

ahh! make sense

done